VCFv4.3.tex

```diff
@@ -424,16 +427,17 @@ \subsubsection{Genotype fields}
   No white-space or semi-colons permitted.
   \item GQ (Integer): Conditional genotype quality, encoded as a phred quality $-10log_{10}$ p(genotype call is wrong, conditioned on the site's being variant).
   \item GP (Float): Genotype posterior probabilities in the range 0 to 1 using the same ordering as the GL field; one use can be to store imputed genotype probabilities.
-  \item GT (String): Genotype, encoded as allele values separated by either of $/$ or $\mid$.
+  \item GT (String): Genotype, encoded as allele values followed by either of $/$ or $\mid$. 
+  The last separator may be ommitted if un-needed.
```

Comment:
We need to explicitly state what implied value of the last separator is. `1/1` could be `1/1/` or it could be `1/1|`. IMO it should default to the type of the final separator that is actually specified.

Reply:
hmmm.  I think that 0/1|2 means that the 0 & 2 are not phased and that only the 1 is phased. this is in agreement with 0|1/2 in which the 0 would be phased and the 1 & 2 are not. but I agree that 0|1|2 means that they are phased, so perhaps we say that if unspecified, the last haplotype is phased iff all the separators are | (which would also mean that in a mathematical sense a haploid is always phased...which is of course true)